### DISCUSSION TITLE
SPITFIRE modes

### DISCUSSION THREAD

**Discussion by ftzahura** [NGEET/fates#1159] [Q&A]
Hi all,

I am currently working on prescribing historical fire events using FATES-SPITFIRE module. According to the user guide, setting fates_spitfire_mode =3 uses an external confirmed ignition dataset. However, I am unable to figure out how to provide this external dataset to FATES for the gridcells in a burn area. From the fire module, I found this option sets currentSite%FDI = 1.0_r8 and cloud_to_ground_strikes = 1.0_r8. Is this the step assigning the external confirmed ignition?

Thank you,
Faria

**Comment by pollybuotte** [comment]
Hi Faria,

I'm attaching a file that I used to prescribe ignitions according to historical burn area in the Sierra Nevada Mountains of California.  It is on the same grid as my domain and surface data files. Hope this example helps!
[SN_prescribed_ignitions_1perday_update.cdl.zip](https://github.com/NGEET/fates/files/14316588/SN_prescribed_ignitions_1perday_update.cdl.zip)

**Comment by ftzahura** [comment]
Hi Polly,

Thank you so much for sharing the prescribed ignition data. I explored the dataset. It seems like some of the grids in the study domain,  are assigned number of ignitions (lnfm) = 0.03 number/km2/hr on July 1 for each year between 1908 and 2000. Does this mean on this particular date each year, the grids with lnfm=0.03 experienced fire? The other grids have a high number of 9.969209968386869e+36 associated with them on July 1, which I suppose is an arbitrary value and do not represent burn grids. 

Could you please share the namelist that was used to pass this dataset to FATES for prescribing burn areas?

Thank you,
Faria

**Reply by pollybuotte** [reply]
This was a case where I prescribed the location and frequency of lightning strikes, not actual ignitions.  Sorry for misreading your post. Maybe @slevis-lmwg you an example of the prescribed successful ignitions file?

**Reply by slevis-lmwg** [reply]
I see comments (that I probably wrote) pertaining to such a file here:
`<your_ctsm_path>/cime_config/testdefs/testmods_dirs/clm/FatesFireLightningPopDens/README`
I have no sense of whether these comments remain up-to-date.
@lmkueppers may have newer information.
Also I likely still have a copy of the file mentioned in the README, but Lara would have to let me know whether it's ok to share.

**Reply by ftzahura** [reply]
@slevis-lmwg Thanks for sharing the file path. I found the README file and the instruction is helpful. If possible, I'd appreciate access to the ignition file. Otherwise, if I could get a sense of the file structure for 'ignition_1980_to_2016_monthly_20190801.nc', I can work on creating an ignition dataset for my study region. Thank you!

**Reply by slevis-lmwg** [reply]
@glemieux offered to check in with Lara about possibly making this data available.

**Reply by ftzahura** [reply]
Thank you so much! I really appreciate the help.
